catalog/MDCCatalog/AppDelegate.swift

```diff
@@ -92,6 +94,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     //
     // Apply color scheme to UIKit components.
+//    UIView.appearance().backgroundColor = colorScheme.backgroundColor
```

Comment:
I'm late to the party. Commented code?

Reply:
Yep, it's definitely a mess. This is the PR used for an internal design review yesterday.  The focus is on the color scheme and the binding to components in the "Experimental" theme methods.  The catalog/example code was hackily disabled to test that UIAppearance works correctly on components.